config/templates/operator.yaml.in

```diff
@@ -0,0 +1,302 @@
+# Copyright 2021 The Cockroach Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# {{ .GeneratedWarning }}
+#
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
So the RBAC is actually already generated, but we are not using it correctly. Can we use https://github.com/cockroachdb/cockroach-operator/blob/master/config/rbac/role.yaml and then patch it with the extra things we need?  I am also wondering if we should have a separate RBAC file.  @udnay thoughts?

Reply:
Can you elaborate on what the "extra tings" are? Can role.yaml just be merged in?

Reply:
I think we can address this in a separate PR/issue. At least we don't regress with this PR.

Any objections?

Reply:
Nope, I think that's fine